src/component/header.js

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import Styled from 'styled-components';
+import FontAwesome from 'react-fontawesome';
+
+import Themeline from './shared/themeline';
+import Scrible from '../assets/images/logos/scribble logo black-03.svg';
+import Sense from '../assets/images/logos/sense logo black -03.svg';
+import Texual from '../assets/images/logos/textual logo black -03.svg';
+
+const Container = Styled.div`
+  height: 35vh;
+  display: flex;
+  padding-bottom: 4vh;
+`;
+
+const LogoContainer = Styled.div`
+  height: 220px;
+  border-right-style:  solid;
+  border-width: 1px;
+  display: flex;
+  flex: 0.4;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-left: 2vw;
+  padding-right: 2vw;
+  margin-top: auto;
+  margin-bottom: auto;
+`;
+
+const MenuContainer = Styled.div`
```

Comment:
I can see repetition in the bottom components. Check this: https://github.com/styled-components/styled-components/issues/954 And other methods available.